### PR TITLE
Doc updates

### PR DESCRIPTION
**Pull request by dowen12**
*Description of changes:*
Update for Wireshark

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: docs/partner_editable/additional_info.adoc [hunk 1]
```diff
@@ -38,12 +38,10 @@ name and password are the tooladmin credentials you submitted.
 .. Ntopng: Connect to the administrator console using the link listed on the
 output of the CloudFormation stack, http://elbaddress:3000. The user
 name and password are the default Ntopng credentials.
-.. Wireshark: Connect by using SSH to the instance with ssh
-ubuntu@ipaddress. Set the password for tooladmin using sudo passwd
-tooladmin. This enables remote desktop protocol (RDP) access for the
-defined user, which is used to connect to the Domain Name System of the
-Wireshark ELB instance. Then start Wireshark from the UI using sudo
-wireshark. Monitor the nurx0 interface for decrypted cloud traffic.
+.. Wireshark: Connect to the wireshark console using the link listed on the
```
Comment (annaone) [inline docs/partner_editable/additional_info.adoc:41]: Testing comments for doc review

Comment (dowen12) [reply docs/partner_editable/additional_info.adoc:41]: thanks for the update

File: docs/partner_editable/additional_info.adoc [hunk 2]
```diff
@@ -38,12 +38,10 @@ name and password are the tooladmin credentials you submitted.
...
 output of the CloudFormation stack, http://elbaddress:3000. The user
 name and password are the default Ntopng credentials.
-.. Wireshark: Connect by using SSH to the instance with ssh
-ubuntu@ipaddress. Set the password for tooladmin using sudo passwd
-tooladmin. This enables remote desktop protocol (RDP) access for the
-defined user, which is used to connect to the Domain Name System of the
-Wireshark ELB instance. Then start Wireshark from the UI using sudo
-wireshark. Monitor the nurx0 interface for decrypted cloud traffic.
+.. Wireshark: Connect to the wireshark console using the link listed on the 
+output of the CloudFormation stack, https://elbaddress:14500. The user name and
```
Comment (annaone) [inline docs/partner_editable/additional_info.adoc:42]: another test comment

**Comment by annaone** [comment]
Looks good!

**Pull request merged**